app/pages/_app.tsx

```diff
@@ -4,34 +4,34 @@ import Head from "next/head";
 import "../styles/globals.scss";
 
 const metaTags = {
-  title: "Auckland DHB Vaccination Rates to 90%",
+  title: "Aotearoa NZ Vaccination Rates to 90% - Outoflockdown.co",
```

Comment:
Not sure how we should word the title + description. Appreciate any feedback

Reply:
I like it one small suggestion:

```suggestion
  title: "Aotearoa NZ Vaccination Rates to 90% - outoflockdown.co",
```

Reply:
Maybe `New Zealand` instead of `NZ` too?